feature-utils/poly-analysis/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@polypoly-eu/poly-analysis",
+  "main": "src/index.js",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "rollup -c rollup.config.js"
+  },
+  "dependencies": {
+    "react": "^17.0.2",
+    "@polypoly-eu/poly-import": "file:../../feature-utils/poly-import"
```

Comment:
So, we have `poly-analysis` depend on `poly-import`... That's not how things should be, **but**, I think it's an acceptable intermediate state. But then let's use it! No need to duplicate stuff like the `Telemetry` class if we can just use it in _poly-analysis_ from _poly-import_, and so on.

Reply:
That wasn't the case initially, but seeing how we needed the `ZipFile `class for `runAnalysis`, we had two options, one was to pass the class as an argument (which was done initially) and the other was to import it in. It does feel weird importing one package to the other, that's why we refrained from that in the beginning and duplicated these classes. For now should we go back to passing `ZipFile` as an argument and remove the dependency on poly-import, or should we commit to the dependency and remove the duplicate files?